README.md

```diff
@@ -186,58 +153,49 @@ specifically.
 docker build --build-arg arch=arm64 .
 ```
 
-### Automated Builds
+## Automated Builds
 
-#### Linux auto build @ TravisCI
+### Linux auto build @ TravisCI
 
  * [Build](https://travis-ci.org/htm-community/nupic.cpp)
  * [Config](./.travis.yml)
 
-#### Mac OS/X auto build @ CircleCI
+### Mac OS/X auto build @ CircleCI
 
  * [Build](https://circleci.com/gh/htm-community/nupic.cpp/tree/master)
  * [Config](./.circleci/config.yml)
  * Local Test Build: `circleci local execute --job build-and-test`
 
-#### Windows auto build @ AppVeyor
+### Windows auto build @ AppVeyor
 
  * [Build](https://ci.appveyor.com/project/htm-community/nupic-cpp/branch/master)
  * [Config](./appveyor.yml)
 
-#### ARM64 auto build @ CircleCI
+### ARM64 auto build @ CircleCI
 
 This uses Docker and QEMU to achieve an ARM64 build on CircleCI's x86 hardware.
 
  * **TODO!** [Build]()
  * [Config](./.circleci/config.yml)
  * Local Test Build: `circleci local execute --job arm64-build-test`
 
+# Using a Graphical Integrated Development Environment
 
-### Testing
-
-#### Unit tests for the library
-
-There are two sets of unit tests.
- * C++ Unit tests -- to run: `cd build/Release/bin; ./unit_tests`
- * Python Unit tests -- to run: `python setup.py test`
-
-### Using graphical interface
-
-#### Generate the IDE solution  (Netbeans, XCode, Eclipse, KDevelop, etc)
+## Generate the IDE solution  (Netbeans, XCode, Eclipse, KDevelop, etc)
 
  * Choose the IDE that interest you (remember that IDE choice is limited to your OS).
  * Open CMake executable in the IDE.
  * Specify the source folder (`$HTM_CORE`) which is the location of the root CMakeList.exe.
  * Specify the build system folder (`$HTM_CORE/build/scripts`), i.e. where IDE solution will be created.
  * Click `Generate`.
 
-#### For MS Visual Studio 2017 as the IDE
+## For MS Visual Studio 2017 as the IDE
  * Double click startupMSVC.bat  -- This will setup the build and create the solution file (.sln).
  * Double click build/scripts/htm.cpp.sln -- This starts up Visual Studio
  * In the solution explorer window, right Click on 'unit_tests' and select `Set as StartUp Project` so debugger will run unit tests.
  * Start a debug session.
```

Comment:
Hi DKeeney,
I Just notice  that the MSVC instructions are duplicated in two places:
1) In the section "Installation"
2) In the section "Using a Graphical Integrated Development Environment"

Can we remove the MSVC instructions from the IDE section, and consolidate them into just the installation instructions section?

Reply:
We can add a hyper link from the IDE section to the installation instructions section like: https://github.com/htm-community/nupic.cpp#simple-build-on-windows-ms-visual-studio-2017

Reply:
I made this change.  You can preview the README as it will be rendered by github.com here:
https://github.com/htm-community/nupic.cpp/blob/72685176bdd7e5fe5abf50f0674c538fd06701dd/README.md#for-ms-visual-studio-2017-as-the-ide